src/goto-checker/properties.h

```diff
@@ -73,7 +73,7 @@ struct property_infot
 };
 
 /// A map of property IDs to property infos
-typedef std::unordered_map<irep_idt, property_infot> propertiest;
+typedef std::map<irep_idt, property_infot> propertiest;
```

Comment:
I know there are some performance concerns related to ordered maps. I know this choice was made to "fix" the output order, but maybe this should be checked more carefully since other output does not have this problem?

Reply:
I have consulted with @peterschrammel offline and this doesn't appear to be a problem.